bundles/org.eclipse.ui.editors/src/org/eclipse/ui/internal/texteditor/stickyscroll/StickyScrollingHandler.java

```diff
@@ -122,9 +122,7 @@ private StickyScrollingControlSettings loadControlSettings(IPreferenceStore stor
 		int stickyScrollingMaxCount= store.getInt(EDITOR_STICKY_SCROLLING_MAXIMUM_COUNT);
 
 		Color lineNumberColor= new Color(PreferenceConverter.getColor(store, EDITOR_LINE_NUMBER_RULER_COLOR));
-		sourceViewer.getTextWidget().addDisposeListener(e -> lineNumberColor.dispose());
```

Comment:
It's a little confusing to see this change mixed into a PR about AbstractUIPlugin.  Maybe this is accidental and meant to be in a different PR about not disposing colors?

Reply:
Sorry, forgot to rebase onto latest master. The already merged change is not removed from this PR.